src/Native/Runtime/MathHelpers.cpp

```diff
@@ -82,7 +82,6 @@ EXTERN_C REDHAWK_API float REDHAWK_CALLCONV RhpFltRound(float value)
 
 #endif // CORERT
 
-#ifdef _ARM_
```

Comment:
Could you please add the C# implementation under `#if ARM` instead? Removing this ifdef is problematic for the closed part of .NET Native for UWP.